shared/js/background/email-utils.js

```diff
@@ -91,13 +89,6 @@ export const getAddresses = () => {
     }
 }
 
-function sendPixelRequest (pixelName, params = {}) {
```

Comment:
Would be nice to split this part of the change out, but I don't insist if you'd rather not.